schemas/atlas/parcellationEntityVersion.schema.tpl.json

```diff
@@ -80,7 +85,7 @@
     },
     "versionInnovation": {
```

Comment:
it might be a good idea to also make this an optional property in the research products.